codeship-steps.yml

```diff
@@ -6,5 +6,13 @@
   steps:
     #-  command: ./.ci/travis/travis.sh site
     #-  command: ./.ci/travis/travis.sh nondex
-    -  command: ls -la
-    -  command: .ci/run-link-check-plugin.sh
+    - command: |
+        ls -la
+        SKIP_FILES1=".github|distelli-manifest.yml|fast-forward-merge.sh"
+        SKIP_FILES2="LICENSE|LICENSE.apache20|README.md|release.sh|RIGHTS.antlr"
+        SKIP_FILES3="shippable.yml|wercker.yml|appveyor.yml|circle.yml"
+        SKIP_FILES="$SKIP_FILES1$SKIP_FILES2$SKIP_FILES3"
```

Comment:
The rightmost item should be separated from the leftmost on the next line:
SKIP_FILES="$SKIP_FILES1`|`$SKIP_FILES2`|`$SKIP_FILES3"
 OR
SKIP_FILES1=".github`|`distelli-manifest.yml`|`fast-forward-merge.sh`|`" (and so on)